docs/extenders/SecureCredentials.md

```diff
@@ -18,13 +19,3 @@ Initialize the `KeytarApi()`.
        "@zowe/zowe-explorer-api": "1.16.0"
    }
    ```
-
-Get access to the initialized Zowe Explorer API objects that are provided by VS Code during or after activation. Use the following code snippet to gain access to `KeytarApi()`:
-
-```typescript
-export function activate(context: vscode.ExtensionContext) {
-  const log = imperative.Logger.getAppLogger();
-  const keytarApi = new KeytarApi(log);
-  await keytarApi.activateKeytar(imperative.CredentialManagerFactory.initialized, EnvironmentManager.isTheia());
-}
-```
```

Comment:
I might be confusing myself, but I believe this was the snippet of code we wanted to keep.
So, the current steps 2 and 3 could be replaced with:

2. Use the following code snippet to gain access to `KeytarApi()`:

```typescript
export function activate(context: vscode.ExtensionContext) {
  const log = imperative.Logger.getAppLogger();
  const keytarApi = new KeytarApi(log);
  await keytarApi.activateKeytar(imperative.CredentialManagerFactory.initialized, EnvironmentManager.isTheia());
}
```

Reply:
I thought it was the other way around???? i'd swear someone said 'just the three lines of code"

Reply:
@JamesBauman, @zFernand0 is correct in this, we should remove the 1,2,&3 and keep the section with the activate code snippet